services/wireguard/wgcfg/device_config.go

```diff
@@ -38,12 +38,13 @@ type Stats struct {
 
 // DeviceConfig describes wireguard device configuration.
 type DeviceConfig struct {
-	IfaceName  string    `json:"iface_name"`
-	Subnet     net.IPNet `json:"subnet"`
-	PrivateKey string    `json:"private_key"`
-	ListenPort int       `json:"listen_port"`
-	DNSPort    int       `json:"dns_port,omitempty"`
-	DNS        []string  `json:"dns"`
+	IfaceName    string `json:"iface_name"`
+	WireguardMTU int
```

Comment:
I would name it without `wireguard` word, since it's already a wireguard device config.
Do we need to set `json:"mtu"`?

Reply:
OK